src/rest-client.ts

```diff
@@ -599,6 +602,14 @@ export class RestClient extends BaseRestClient {
     return this.postPrivate('/api/v5/account/set-position-mode', { posMode });
   }
 
+  setAccountMode(acctLv: AccountLevel): Promise<AccountModeResult[]> {
+    return this.postPrivate('/api/v5/account/set-account-level', { acctLv });
+  }
+
+  setAutoLoan(autoLoan: boolean): Promise<AutoLoanResult[]> {
+    return this.postPrivate('/api/v5/account/set-auto-loan', { autoLoan });
+  }
+
```

Comment:
```suggestion
  setAutoLoan(params: { autoLoan: boolean; }): Promise<AutoLoanResult[]> {
    return this.postPrivate('/api/v5/account/set-auto-loan', params);
  }
  
```

It's safer to use object notation for everything, even if this request only has one param right now. From experience, exchanges can add more parameters in future (some optional & some required), so it can get problematic to update these methods in future if they don't already use objects as params.

Same for setAccountMode please